lighthouse-core/test/audits/long-tasks-test.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const LongTasks = require('../../audits/long-tasks.js');
+const createTestTrace = require('../create-test-trace.js');
+const devtoolsLog = require('../fixtures/traces/progressive-app-m60.devtools.log.json');
+
+const TASK_URL = 'https://pwa.rocks/';
+
+/* eslint-env jest */
+
+/**
+ * @param {Number} count
+ * @param {Number} duration
+ * @param {Boolean} withChildTasks
+ */
+function generateTraceWithLongTasks(
+  count,
+  duration = 200,
+  withChildTasks = false
+) {
+  const baseTs = 1000;
+  const traceTasks = [];
+  for (let i = 1; i <= count; i++) {
+    const ts = baseTs * i;
+    const task = {ts, duration};
+    task.children = [];
+    if (withChildTasks) {
+      task.children.push({
+        ts: ts + 10,
+        duration: duration / 3,
+        url: TASK_URL,
+      });
+      task.children.push({
+        ts: ts + duration / 2,
+        duration: duration / 3,
+        url: TASK_URL,
+      });
+    }
+    traceTasks.push(task);
+  }
+  return createTestTrace({
+    topLevelTasks: traceTasks,
+  });
+}
+
+describe('Long tasks audit', () => {
+  it('should pass and be non-applicable if there are no long tasks', async () => {
+    const artifacts = {
+      traces: {defaultPass: generateTraceWithLongTasks(0)},
+      devtoolsLogs: {defaultPass: devtoolsLog},
+    };
+    const result = await LongTasks.audit(artifacts, {computedCache: new Map()});
+    expect(result.score).toBe(1);
+    expect(result.details.items).toHaveLength(0);
+    expect(result.displayValue).toBeUndefined();
+    expect(result.notApplicable).toBeTruthy();
+  });
+
+  it('should return a list of long tasks with duration >= 50 ms', async () => {
+    const artifacts = {
+      traces: {defaultPass: generateTraceWithLongTasks(4)},
+      devtoolsLogs: {defaultPass: devtoolsLog},
+    };
+    const result = await LongTasks.audit(artifacts, {computedCache: new Map()});
+    expect(result.score).toBe(0);
+    expect(result.details.items).toHaveLength(4);
+    expect(result.displayValue).toBeDisplayString('4 long tasks found');
+    expect(result.notApplicable).toBeFalsy();
+
+    for (const item of result.details.items) {
+      expect(Number.isFinite(item.duration)).toBeTruthy();
+      expect(item.duration).toBeGreaterThanOrEqual(50);
```

Comment:
```suggestion
      expect(item.duration).toEqual(200);
```

at least one explicit test for not using multiplier :)